### PR TITLE
Enable experimental packaging test permanently

### PR DESCRIPTION
**Pull request by larioj**

### CODE REVIEW

**Comment by tamarrow** [comment]
@larioj please add info on what a developer needs to make these tasks pass in the README. Can you also remove the envvar from the CIs please?

File: README.rst [hunk 1]
```diff
@@ -114,13 +114,16 @@ to test SSL certs. To run this test suite be sure to add this resolution to your
...
-Tox will run unit and integration tests in Python 3.4 using a temporarily
+Tox will run unit and integration tests in Python 3.5 using a temporarily
 created virtualenv.
 
 You can set :code:`DCOS_CONFIG` to a config file that points to a DC/OS
 cluster you want to use for integration tests. This defaults to
 :code:`~/.dcos/dcos.toml`
 
+Note that in order for all the integration tests to pass, your DC/OS cluster
+must have the experimental packaging features enabled.
```
Comment (tamarrow) [inline README.rst]: How does one enable these?

File: README.rst [hunk 2]
```diff
@@ -114,13 +114,16 @@ to test SSL certs. To run this test suite be sure to add this resolution to your
 Running
 #######
 
-Tox will run unit and integration tests in Python 3.4 using a temporarily
+Tox will run unit and integration tests in Python 3.5 using a temporarily
```
Comment (tamarrow) [inline README.rst]: Please revert these, they are being updated in #878

**Comment by larioj** [comment]
@tamarrow The CI is updated. I could link this: https://docs.mesosphere.com/1.9/administration/installing/custom/configuration-parameters/
and point the audience to the cosmos_config parameters staged_package_storage_uri and package_storage_uri.

Would that be fine?

There is a blog post that goes into detail about enabling the features. But sadly that is not live yet. And will not be for a while.

**Comment by tamarrow** [comment]
@larioj Is there an open version equivalent?

**Comment by larioj** [comment]
@tamarrow An open version equivalent of what? The link? The parameters? The blog post?

**Comment by tamarrow** [comment]
the doc link. The one you posted is for the Enterprise version, but this is an open repo :)

**Comment by larioj** [comment]
@tamarrow Yes https://dcos.io/docs/1.9/administration/installing/custom/configuration-parameters/

**Pull request merged**